src/Exception/EntityNotFoundException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\Common\Exception;
+
+use Throwable;
+
+class EntityNotFoundException extends ClientException
+{
+    public function __construct(
+        string $token = ExceptionTokens::ENTITY_NOT_FOUND,
+        string $message = ExceptionTokens::ENTITY_NOT_FOUND,
```

Comment:
https://github.com/LinioIT/bob-for-alice/blob/master/src/Order/OrderService.php#L185 - here's an example that actually requires `$message` for more details internally though. I'm thinking of maybe just changing `$message` to `$internalDetail` or something.

What do you think?